### PR TITLE
Radio - Bordered Variant (React)

### PR DESCRIPTION
**Pull request by anechol**
## Description

Adds bordered radio variant to React docs and fixes `customContent` bug.

[Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%F0%9F%A7%A9-Sage-components?node-id=15048%3A32130)

## Screenshots

Screen Shot 2022-10-03 at 10 21 01 AM

## Testing in `sage-lib`

- View [Radio](http://localhost:4100/?path=/docs/sage-radio--default)
- Check that bordered variant matches styles in Figma

## Testing in `kajabi-products`

N/A

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼 - thanks so much for the quick turnaround!!

File: packages/sage-react/lib/Toggle/Radio.story.jsx [hunk 1]
```diff
@@ -22,6 +22,53 @@ const Template = (args) => <Radio {...args} />;
 
 export const Default = Template.bind({});
 
+export const BorderedRadio = (args) => {
```
Comment (monkeypox8) [inline packages/sage-react/lib/Toggle/Radio.story.jsx:25]: (non-blocking)
Could we also add the following to the default story's args list? Would better expose the prop for use & clean up the console errors:
```
    id: 'radio-1',
    name: 'radio-1',
    hasBorder: false,
```

**Review by monkeypox8 — APPROVED** [review]
🔥 👍 LGTM!

Added one non-blocking suggestion to help ease some of the confusion around our documentation.

**Review by ju-Skinner — APPROVED** [review]
LGTM.

File: packages/sage-react/lib/Toggle/Radio.story.jsx (hunk 1 above)
Comment (anechol) [reply packages/sage-react/lib/Toggle/Radio.story.jsx:25]: Added these, thanks.

**Pull request merged**